releasenotes/notes/circuit-from-instructions-832b43bfd2bfd921.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    Added the methods `~.QuantumCircuit.from_instructions` and
+    `~.QuantumCircuit.from_instruction_tuples` for constructing quantum circuits
```

Comment:
I think these two methods got combined.  The cross-ref syntax you want here is `` :meth:`.QuantumCircuit.from_instructions` `` - the `:meth:` lead-in is what triggers the Sphinx lookup, and the `~` (which I've omitted) would cause the display to be truncated down to `from_instructions()` instead of `QuantumCircuit.from_instructions()`.

Reply:
Done.